manuscript/Figure3-calculations.jl

```diff
@@ -36,12 +36,12 @@ for oldname in names(mdata_ecs_df)[15:2414]
     rename!(mdata_ecs_df, oldname => randstring(['A':'Z'; 'a':'z'], 12))
 end
 newnames = names(mdata_ecs_df)[15:2414]
-open("longnames.txt", "w") do io
+open(scratchfiles("longnames.txt"), "w") do io # @Hugemiler I can't figure out where these are used
```

Comment:
But there's no code that references these files... 

![image](https://user-images.githubusercontent.com/3502975/228633958-4eccf2fc-83f1-40c6-be1a-d9fcbe3b5d15.png)

![image](https://user-images.githubusercontent.com/3502975/228634114-1b215b02-ac4b-4637-a9f4-4f84e526d829.png)

Reply:
It's because I took it away from the markdown when we decided not to include ECs on Figure 3, but you can see the `hashnamestable` argument is implemented on `weighted_hpimportances` on the picture you sent. I used to build this table and give it to the function whenever I analyzed ECs.

Reply:
And to build this table, I consume `longnames.txt` and `hashnames.txt`

Reply:
Got it. There's a better way to do this, but we'll leave it for now.